### PR TITLE
core.check_update: Add config documentation for check_update core config

### PR DESCRIPTION
**Pull request by PuneethaPai**
Fixes: Add doc to changes for Issue https://github.com/iterative/dvc/issues/1894

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
> Add doc to changes for Issue iterative/dvc#1894

So. iterative/dvc/pull/4334 ? 🙂

File: content/docs/command-reference/config.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ This is the main section with the general config options:
   and `false` (default). Set with the `--no-scm` option of `dvc init`
   ([more details](/doc/command-reference/init#initializing-dvc-without-git)).
 
+- `core.check_update` - used to enable/disable dvc checking for update and
+  notifying user. Accepts values `true` (default) and `false`.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/config.md:91]:
```suggestion
- `core.check_update` - disable/enable DVC's automatic update checks, which
  notify the user when a new version is available. Accepts values `true` (default)
  and `false`.
```

File: content/docs/command-reference/config.md [hunk 2]
```diff
@@ -87,6 +87,10 @@ This is the main section with the general config options:
   and `false` (default). Set with the `--no-scm` option of `dvc init`
   ([more details](/doc/command-reference/init#initializing-dvc-without-git)).
 
+- `core.check_update` - disable/enable DVC's automatic update checks, which
+  notify the user when a new version is available. Accepts values `true` (default)
+  and `false`.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/config.md:92]:
```suggestion
- `core.check_update` - disable/enable DVC's automatic update checks, which
  notify the user when a new version is available. Accepts values `true`
  (default) and `false`.
```

**Comment by jorgeorpinel** [comment]
@PuneethaPai please try to remind us to merge this when iterative/dvc/pull/4334 is merged (or to review again if there are more changes). Thanks!

**Comment by efiop** [comment]
@jorgeorpinel Core PR is merged. :slightly_smiling_face:

**Pull request merged**